frontend/src/sass/pages/_meet-the-team.scss

```diff
@@ -16,11 +16,28 @@ $team-individual-img-height: 150px;
 
 .team {
 	
+	display: flex;
+	
+	&__first-column {
```

Comment:
Could we please utilise the responsive grid we have from foundation instead of creating out own columns?